staging/src/k8s.io/api/testdata/HEAD/autoscaling.v2.HorizontalPodAutoscaler.json

```diff
@@ -279,4 +279,4 @@
       }
     ]
   }
-}
+}
```

Comment:
Is this something we care about?

I ran yamlfmt on these, for kicks, and it complained.

Reply:
this is the output of our encoder, so I'd rather use it verbatim